src/category_theory/full_subcategory.lean

```diff
@@ -26,19 +26,21 @@ section induced
 
 -/
 
-variables {C : Type u₁} {D : Type u₂} [𝒟 : category.{v} D]
+variables {C : Type u₁} (D : Type u₂) [𝒟 : category.{v} D]
```

Comment:
I'm surprised you want to make `D` an explicit argument. Shouldn't it normally be inferable from `F`? Are there specific places where this change is needed?